Tools-Override/FrameworkTargeting.targets

```diff
@@ -228,11 +228,18 @@
 
   <Target Name="BinPlaceFiles"
           Condition="'@(BinPlaceDir)' != ''"
-          DependsOnTargets="GetBinPlaceItems" >
+          DependsOnTargets="GetBinPlaceItems"
+          Inputs="@(BinPlaceDir);%(BinPlaceDir.ItemName)"
```

Comment:
That's hilarious, I can't remember anything anymore. I probably created that error message. haha